### DISCUSSION TITLE
Can't process my application for Global Campus

### DISCUSSION THREAD

**Discussion by zytoh0** [community/community#50423] [GitHub Education]
I tried to submit my application for Global Campus by clicking on the "Process my application" as shown in the screenshot below, and I received the following error:

Your application cannot be reviewed until you fix the following:
You are significantly more likely to be verified if you have completed your [GitHub user profile](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/personalizing-your-profile) with your full name as it appears in your academic affiliation document plus a short bio.

As you can see from my profile, I've already completed my profile with my full name plus a short bio. I've tried with all kinds of documents including current status of letter, student ID all of them are dated. Anyone with solutions?

<img width="754" alt="Screenshot 2023-03-19 at 8 32 16 PM" src="https://user-images.githubusercontent.com/90326544/226177423-044faa2a-aeda-4277-82ab-888181ab4615.png">

**Comment by zyq-20000901** [comment]
I have the same question！
